analyze_map_test.go

```diff
@@ -178,13 +178,13 @@ func TestAnalyzeMapBody(t *testing.T) {
 		},
 	}
 
-	for name, tc := range testcases {
-		tc := tc
+	for name, test := range testcases {
+		testcase := test
```

Comment:
Suggest you use the same variable name to make it impossible to accidentally use the variable that gets reassigned in the loop. I think the short variable name `tc` was a well-enough understood pattern that you should just leave it.  Assume I'd leave the same comment else where the variable name was changed.

It seems there wasn't a reason to modify this file.